specification/schemas/HookActionApproveReturn.json

```diff
@@ -0,0 +1,30 @@
+{
+  "type": "object",
+  "required": [
+    "action_type",
+    "values"
+  ],
+  "additionalProperties": false,
+  "properties": {
+    "action_type": {
+      "type": "string",
+      "pattern": "^approve-return$",
+      "example": "approve-return"
+    },
+    "values": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "required": [
+          "return_id"
```

Comment:
I do not understand why would return_id be an accepted value here. We are applying these actions on any return resource that matches the predicates.

Reply:
Yep, the entire `values` part can be dropped. It will approve/reject the (return) resource that triggered the hook.

Reply:
fixed